lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -60,6 +60,7 @@ def initialize
         @rewrite_not_or_regular_expression = false
         @rewrite_and_not_operator = false
         @debug_rewrite = false
+        @execution_query_rate = 1.0
```

Comment:
"execution_query" is a bit strange.
How about `keep_ratio`, `filter_rate` (meaning is inverted) or `omit_rate` (meaning is inverted)?

Reply:
I see.
I used `omit_rate` on this option name.